src/reddit_analysis.py

```diff
@@ -0,0 +1,48 @@
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+
+
+class RedditAnalysis:
+    def __init__(self) -> None:
+        self.results = {}
+        self.stock_count = {}
+        
+    def run_analysis(self, dict):
+        analyzer = SentimentIntensityAnalyzer()
+
+        for sentence in dict:
+            self.sentiment_analyzer_scores(dict[sentence], analyzer)
+        
+        for key in dict.keys():
+            test = dict.get(key)
```

Comment:
you only use key to get the value, why not use for value in  dict.values() instead of dict.keys()